src/lenskit/data/builder.py

```diff
@@ -165,8 +169,9 @@ def add_relationship_class(
         self,
         name: str,
         entities: RelationshipEntities,
-        allow_repeats: bool = True,
+        repeats: bool = True,
```

Comment:
Then name and behavior of this option are part of a stable API, so we need to keep them the same.